.circleci/config.yml

```diff
@@ -103,11 +100,11 @@ jobs:
                 name: Run integration tests
                 command: |
                     . venv/bin/activate
-                    pip install --upgrade pip
-                    pip install -r tests/requirements.txt
                     npm ci
                     npm run build
-                    pip install -e .
+                    python setup.py sdist
+                    cd dist
+                    find . -name "*.gz" | xargs pip install --no-cache-dir --ignore-installed && cd ..
```

Comment:
Using the opportunity to install from tarball instead of the local folder `-e .`. This change helps identify issues when resource paths change and the MANIFEST.in file isn't synced.